sections/products.liquid

```diff
@@ -0,0 +1,103 @@
+                <path d="M6.35156 5.5918V5.24023C6.35156 4.65866 5.87845 4.18555 5.29688 4.18555C4.7153 4.18555 4.24219 4.65866 4.24219 5.24023V5.94336C4.24219 6.52585 4.71438 6.99805 5.29688 6.99805H8.13281C8.7153 6.99805 9.1875 6.52585 9.1875 5.94336V5.5918H6.35156Z" fill="#D56939"/>
+                <path d="M9.1875 5.94336V5.5918H6.35156V5.24023C6.35156 4.65866 5.87845 4.18555 5.29688 4.18555V6.99805H8.13281C8.7153 6.99805 9.1875 6.52585 9.1875 5.94336Z" fill="#D14D41"/>
+                <path d="M5.64844 2.4043H5.29688H4.94531L4.24219 2.75586V3.10742C4.24219 3.689 4.7153 4.18555 5.29688 4.18555C5.87845 4.18555 6.35156 3.689 6.35156 3.10742V2.75586L5.64844 2.4043Z" fill="#FFE1BA"/>
+                <path d="M6.35156 3.10742V2.75586L5.64844 2.4043H5.29688V4.18555C5.87845 4.18555 6.35156 3.689 6.35156 3.10742Z" fill="#FFBFAB"/>
+                <path d="M5.29688 2.05273H4.24219V2.75586H5.29688H7.05469V2.05273H5.29688Z" fill="#474F54"/>
+                <path d="M5.29688 2.05273H7.05469V2.75586H5.29688V2.05273Z" fill="#32393F"/>
+                <path d="M10.1564 7.77978C10.1176 7.50615 10.0406 6.96713 9.8872 5.89355L8.48438 5.24023V8.05273C8.48438 8.60734 8.0066 9.10742 7.40625 9.10742H7.05469V11.2168H7.75781H11.2969L12 10.8652V10.1621C12 9.03739 11.2241 8.05686 10.1564 7.77978Z" fill="#D14D41"/>
+                <path d="M10.5938 6.29492H9.53906C8.95748 6.29492 8.48438 5.82181 8.48438 5.24023C8.48438 4.65866 8.95748 4.18555 9.53906 4.18555H10.5938V6.29492Z" fill="#D56939"/>
+                <path d="M10.3478 9.1181L10.2456 9.50876L9.65145 9.21689C8.9734 9.4609 8.48438 10.1042 8.48438 10.8652C8.48438 11.8344 9.27298 12.623 10.2422 12.623C11.2114 12.623 12 11.8344 12 10.8652C12 9.93221 11.267 9.17398 10.3478 9.1181Z" fill="#32393F"/>
+                <path d="M10.2422 9.81055C9.66061 9.81055 9.1875 10.2837 9.1875 10.8652C9.1875 11.4468 9.66061 11.9199 10.2422 11.9199C10.8238 11.9199 11.2969 11.4468 11.2969 10.8652C11.2969 10.2837 10.8238 9.81055 10.2422 9.81055Z" fill="#DFE7F4"/>
+                <path d="M10.5902 10.8155L10.3478 9.11811C10.3121 9.11595 10.2783 9.10742 10.2421 9.10742C10.0339 9.10742 9.83683 9.15017 9.65137 9.2169L9.89397 10.915L10.5902 10.8155Z" fill="#C7CFE1"/>
+              </svg>
+              Entrega hoy
+            </span>
+          </div>
+        </div>
+      </div>
+    {% else %}
+      <p>No se encontró información para el producto con ID: {{ section.settings[product_id] }}</p>
+    {% endif %}
+  {% endfor %}
+</div>
+
+{% schema %}
+{
+  "name": "Products",
+  "settings": [
+    {
+      "type": "product",
+      "label": "Product 1",
+      "id": "product_1"
+    },
+    {
+      "type": "product",
+      "label": "Product 2",
+      "id": "product_2"
+    },
+    {
+      "type": "product",
+      "label": "Product 3",
+      "id": "product_3"
+    },
+    {
+      "type": "product",
+      "label": "Product 4",
+      "id": "product_4"
+    }
```

Comment:
It is better to use:
- A collection with the featured products, or
- Section blocks: https://shopify.dev/docs/themes/architecture/blocks/section-blocks

```suggestion
```